packages/editor-ui/src/mixins/nodeHelpers.ts

```diff
@@ -531,6 +531,9 @@ export const nodeHelpers = defineComponent({
 
 			if (nodeType !== null && nodeType.subtitle !== undefined) {
 				try {
+					ExpressionEvaluatorProxy.setEvaluator(
+						useSettingsStore().settings.expressions?.evaluator ?? 'tournament',
+					);
```

Comment:
Why are we setting the evaluator in mixins? I'd expect this to happen at `init()` at `App.vue` or similar central init location.

Reply:
I put it there originally to just get it working but after asking someone more familiar with the UI they said they couldn't think of a better place so I just left it.